evasquez26/angular4-startup
# S2 ONE 44-48 Create and edit course section

## Story / Task

[ONE-44](https://bixlabs.atlassian.net/browse/ONE-44) and [ONE-48](https://bixlabs.atlassian.net/browse/ONE-48)

## What this PR does?

WARNING, endpoint still does not work. Allows to create a course initial data.


### Before 

We could not create a course

### After

We can create a course now

### How to test / reproduce

- Login
- Observe that there's a button that will allow you to create a course
- Clicking the button should take you to create a course inside an organization
- Fill all the fields
- Clicking the submit button should allow you to create a course

### TODOs
- [ ] Implement the correct design
- [ ] Figure out what do we pass to `enrollDuration` field

### Related PRs

N/A

## Migrations

NO


Thanks!
